### PR TITLE
[MNG-6829] Replace any StringUtils#isEmpty(String) and #isNotEmpty(String)

### PR DESCRIPTION
**Pull request by timtebeek**
Part of an ongoing effort to phase out StringUtils.
https://issues.apache.org/jira/browse/MNG-6829

Use this link to re-run the recipe: https://public.moderne.io/recipes/org.openrewrite.java.migrate.apache.commons.lang.IsNotEmptyToJdk?organizationId=QXBhY2hlIE1hdmVu

Review requested of @elharo 

---
Following this checklist to help us incorporate your
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed
       for the change (usually before you start working on it).  Trivial changes like typos do not
       require a JIRA issue. Your pull request should address just this issue, without
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MNG-XXX] SUMMARY`,
       where you replace `MNG-XXX` and `SUMMARY` with the appropriate JIRA issue.
 - [x] Also format the first line of the commit message like `[MNG-XXX] SUMMARY`.
       Best practice is to use the JIRA issue title in both the pull request title and in the first line of the commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will
       be performed on your pull request automatically.
 - [ ] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Review by elharo — APPROVED** [review]
I'm surprised none of these change the imports

**Comment by timtebeek** [comment]
Some of these remove imports though right? But only when there's no other methods being called. There's quite a few `StringUtils.replace` that I'd like to tackle next, which should help remove further imports.

I'd not reconsidered replacing imports yet (going from one string utils implementation to another), as it seemed removing was preferred over replacement. Is that still the case?

We can discuss what to tackle next and how to approach things going forward, as there were other concerns as well such as matching up per project Jira issues. Perhaps we can schedule a brief call or chat at a conference to discuss?

**Comment by elharo** [comment]
I didn't notice any import changes. Maybe I just missed them. I'd expect that in some if not all of these files these would be the only methods used from StringUtils so I expected to see some import removals.

File: maven-core/src/main/java/org/apache/maven/lifecycle/internal/DefaultMojoExecutionConfigurator.java [hunk 1]
```diff
@@ -37,7 +37,6 @@
 import org.apache.maven.project.MavenProject;
 import org.apache.maven.shared.utils.logging.MessageBuilder;
 import org.apache.maven.shared.utils.logging.MessageUtils;
-import org.codehaus.plexus.util.StringUtils;
```
Comment (elharo) [inline maven-core/src/main/java/org/apache/maven/lifecycle/internal/DefaultMojoExecutionConfigurator.java:40]: OK, looks like it is sometimes removing an import. That's good.

**Review by elharo — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #8458

**Pull request merged**